src/libcore/convert.rs

```diff
@@ -213,7 +213,7 @@ pub trait AsMut<T: ?Sized> {
 ///
 /// # Generic Implementations
 ///
-/// - [`From<T>`]` for U` implies `Into<U> for T`
```

Comment:
Please keep this as is and let `From<T>` be linked to `From`.

Reply:
The documentation is inconsistent in this regard.  Some places it uses

    [`From<T>`]` ...`

other times it uses

    [`From`]`<T> ...`

Why do you prefer the former?  The latter reduces copy paste of the link destinations.

Reply:
Otherwise it looks like this: [`From`](https://doc.rust-lang.org/nightly/std/convert/trait.From.html)`<T>` which doesn't look great.

Reply:
See [convert module](https://doc.rust-lang.org/std/convert/index.html#generic-implementations) it looks fine

Reply:
So should I fix [this line](https://github.com/rust-lang/rust/blob/eba03d462e987a969d21aa74632e4d96dbe9751e/src/libcore/convert.rs#L27) too?

Reply:
Personally I think it looks weird; there's a space in-between that shouldn't be there, but I leave it to Steve.

Reply:
We don't have a good convention, and I agree the space is unfortunate. Maybe @rust-lang/rustdoc can weigh in?

Reply:
I tend to prefer how @czipperz wrote it.

Reply:
Although the spacing should theoretically be more messed up in the later as @Centril pointed out, it really looks more messed up the former when rendered by rustdoc.  Strange.

Reply:
This style (of putting `<T>` not inside the link) is also much more popular than the other way around.  Before this pr, its used 40 times throughout the code base vs 10 for the other style.

Here's an example from `Weak`'s documentation: 
![image](https://user-images.githubusercontent.com/10054343/56088376-19cb0280-5e45-11e9-99a8-4722866164f0.png)